MREUnityRuntime/MREUnityRuntimeLib/Assets/AssetLoader.cs

```diff
@@ -363,7 +386,10 @@ internal async void OnCreateAsset(CreateAsset payload, Action onCompleteCallback
             }
             else
             {
-                response.FailureMessage = $"Not implemented: CreateAsset of type {(def.Prefab != null ? "Prefab" : "Mesh")}";
+                if (response.FailureMessage == null)
+                {
+                    response.FailureMessage = $"Not implemented: CreateAsset of new asset type";
```

Comment:
This error message seems a little vague, but I can't come up with anything better.